web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -475,11 +475,11 @@ export default class DrawSupport extends React.Component {
 
         this.removeSnapInteraction();
         if (mapLayerInstance) {
-            switch (mapLayerInstance.type) {
-            case "VECTOR":
+            switch (true) {
+            case (mapLayerInstance.type === "VECTOR"):
```

Comment:
maybe we can define some constants instead of writing fixed strings,
usually this constants are stored in the UTILS file, so in this case they can be created in [DrawSupportUtils](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/openlayers/DrawSupportUtils.js)
```suggestion
            case "VECTOR":
```